airbyte-integrations/connectors/source-snowflake/source_snowflake/schemas/customers.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
What is this file and `airbyte-integrations/connectors/source-snowflake/source_snowflake/schemas/employees.json`?

Reply:
It was a file from the template given by the cdk i will delete it